git-cliff-core/src/release.rs

```diff
@@ -124,15 +133,30 @@ impl<'a> Release<'a> {
 						custom_minor_increment_regex,
 					)?;
 				}
-				let next_version = next_version
-					.increment(
-						&semver?,
-						self.commits
-							.iter()
-							.map(|commit| commit.message.trim_end().to_string())
-							.collect::<Vec<String>>(),
-					)
-					.to_string();
+
+				let next_version = if let Some(bump_as) = &config.bump_as {
+					match bump_as.as_str() {
+						"major" => semver?.increment_major().to_string(),
+						"minor" => semver?.increment_minor().to_string(),
+						"patch" => semver?.increment_patch().to_string(),
```

Comment:
/me thinks if we can use an enum here 🤔 

not a big deal though

Reply:
yup. i made it an enum.